cszczepaniak/cribbly-backend
# implement team controller

# What I added (link any issues addressed)

- A controller which will get and create team objects

# How to test it
Create two users
`curl --request POST --Url "https://localhost:5001/player" -H 'Content-Type: application/json' --data '{\"Email\":\"funke@bluth.com\", \"Name\":\"Tobias Funke\"}'
curl --request POST --Url "https://localhost:5001/player" -H 'Content-Type: application/json' --data '{\"Email\":\"gob@bluth.com\", \"Name\":\"Gob Bluth\"}'`

Create a team by running a POST request, including both the team name, as well as the IDs of the players. Additionally, you can verify that a 400 error will come back if two players are not included in the request body.
`curl --Url "https://localhost:5001/api/team" -H 'Content-Type: application/json' --data '{\"Name\":\"I Blue Myself\", \"Players\":[{\"Id\": 3}, {\"Id\": 4}]}'`

You should now be able to select the players in your local DB and see that they now have the TeamId field updated

To run the GET endpoint, simply include the Id of the team you want at the end of your route

`curl --Url "https://localhost:5001/api/team/31"`

#Overall Note
I have a feeling like there might be a better way to return a Team object that has both players in the Players list property, but what I have should work  (and also work if a three player team ever existed for some reason in the future :))
